src/_C009.ino

```diff
@@ -95,14 +95,14 @@ bool do_process_c009_delay_queue(int controller_number, const C009_queue_element
   String jsonString;
   {
     // Create json root object
-    DynamicJsonBuffer jsonBuffer;
-    JsonObject& root = jsonBuffer.createObject();
+    DynamicJsonDocument jsonBuffer(1024);
+    DynamicJsonDocument root(1024);
     root[F("module")] = String(F("ESPEasy"));
     root[F("version")] = String(F("1.04"));
 
     // Create nested objects
-    JsonObject& data = root.createNestedObject(String(F("data")));
-    JsonObject& ESP = data.createNestedObject(String(F("ESP")));
+    JsonObject data = root.createNestedObject(String(F("data")));
+    JsonObject ESP = data.createNestedObject(String(F("ESP")));
```

Comment:
Why can't this be a reference anymore? (just curious)
At first glance it looks like it will use more memory like this. (I have not looked at the JsonObject code)

Reply:
The library update requires this. https://arduinojson.org/v6/doc/upgrade/